### PR TITLE
logging

### PR DESCRIPTION
**Pull request by chloealee**
#### What's this PR do?
Adds more log messages to debug memory error we're seeing in thor. 

#### How should this be reviewed?
👀 

#### Any background context you want to provide?
More of the convo. [here](https://github.com/DoSomething/rogue/pull/728#issuecomment-396412879)! 

#### Relevant tickets
Continuation of debugging from #728 

#### How to deploy
https://github.com/DoSomething/rogue/blob/master/docs/development/deployments.md

### CODE REVIEW

File: app/Console/Commands/BulkReviewPosts.php [hunk 1]
```diff
@@ -60,13 +60,15 @@ public function handle()
...
         $tags = $this->option('tag') ?? null;
-
+        info('rogue:bulkreviewposts: Beginning to query');
         $posts = Post::setEagerLoads([])
             ->where('campaign_id', $this->argument('campaign'))
             ->where('status', $this->argument('oldStatus'))
             ->where('type', $this->argument('type'))
             ->get();
 
+        info('rogue:bulkreviewposts: Posts result from query: ' . $posts);
```
Comment (sbsmith86) [inline app/Console/Commands/BulkReviewPosts.php]: let's just log the first result `->first()`

Comment (sbsmith86) [reply app/Console/Commands/BulkReviewPosts.php]: also can you limit the query to 10 posts (instead of all of them) has any issues?

Comment (chloealee) [reply app/Console/Commands/BulkReviewPosts.php]: ah yes forgot that! Updated :)

**Comment by DFurnes** [comment]
@chloealee It looks like this query returns quite a lot of results (53,742 rows) on production. It might be a good idea to [chunk the results or use a cursor](https://laravel.com/docs/5.6/eloquent#chunking-results) so we don't need to load all those rows into memory at once.

**Comment by chloealee** [comment]
ohhh that is a great idea @DFurnes ! If we use `chunk` as I added, can we limit the query and dump out things like @sbsmith86 suggested before?

**Comment by sbsmith86** [comment]
@chloealee ooo right right chunking! might be worth seeing if that is all that was needed to deal with the memory issue.

**Comment by DFurnes** [comment]
It might be best to follow @sbsmith86's suggestions first since chunking the query will mess with those things. Once you're confident that we've narrowed it down though, I do think chunking is the right call here.

File: app/Console/Commands/BulkReviewPosts.php [hunk 2]
```diff
@@ -61,30 +61,30 @@ public function handle()
         $log = $this->option('log');
         $tags = $this->option('tag') ?? null;
 
-        $posts = Post::setEagerLoads([])
-            ->where('campaign_id', $this->argument('campaign'))
-            ->where('status', $this->argument('oldStatus'))
-            ->where('type', $this->argument('type'))
-            ->get();
+        Post::where([
+            ['campaign_id', $this->argument('campaign')],
```
Comment (DFurnes) [inline app/Console/Commands/BulkReviewPosts.php]: This is subjective, but I think the way you had the query before (as chained methods) reads a lil' nicer.

**Comment by chloealee** [comment]
ok! So should I revert back to the commit before with the extra log messages and then try chunking?

File: app/Console/Commands/BulkReviewPosts.php (hunk 2 above)
Comment (chloealee) [reply app/Console/Commands/BulkReviewPosts.php]: @DFurnes totally ! I just wasn't sure the syntax for using `chunk` with the chained methods?

Comment (DFurnes) [reply app/Console/Commands/BulkReviewPosts.php]: Same, just instead of using `->get()` to grab the entire result set, call `->chunk(...)` to chunk it!

**Comment by chloealee** [comment]
@sbsmith86 @DFurnes I changed back to more logging and dumping out the first post and if that works, will update again to use the chunking method (@mirie and I chatted about chunking yesterday, oof I totally forgot to add that!)!

**Comment by chloealee** [comment]
lmk if you meant something different though and not by the above!

**Comment by chloealee** [comment]
@sbsmith86 @DFurnes I think that was it and hopefully using `chunk` should work! I just ran on thor and it was successful with the limit of 10 records! 

```
Jun 13 16:11:45 rogue-thor app/run.4976: [2018-06-13 20:11:44] thor.INFO: rogue:bulkreviewposts: Starting to bulk review posts! [] {"user_id":null,"client_id":null,"request_id":null} 
Jun 13 16:11:45 rogue-thor app/run.4976: [2018-06-13 20:11:44] thor.INFO: rogue:bulkreviewposts: Beginning to query [] {"user_id":null,"client_id":null,"request_id":null} 
Jun 13 16:11:45 rogue-thor app/run.4976: [2018-06-13 20:11:44] thor.INFO: rogue:bulkreviewposts: First post result from query: {"id":467915,"signup_id":9162432,"campaign_id":"8064","northstar_id":"594acff2a0bfad6dd410fbbf","type":"text","action":"default","quantity":null,"url":null,"text":"finding the confidence to stick up for myself","status":"pending","source":"sms","source_details":null,"details":null,"created_at":"2018-05-10 17:09:32","updated_at":"2018-05-10 17:09:32","deleted_at":null,"reactions_count":0} [] {"user_id":null,"client_id":null,"request_id":null} 
Jun 13 16:11:47 rogue-thor heroku/run.4976: State changed from starting to up 
Jun 13 16:11:47 rogue-thor heroku/run.4976: Process exited with status 0 
Jun 13 16:11:47 rogue-thor heroku/run.4976: State changed from up to complete 
Jun 13 16:11:54 rogue-thor app/run.4976: [2018-06-13 20:11:47] thor.INFO: rogue:bulkreviewposts: ALL DONE! [] {"user_id":null,"client_id":null,"request_id":null} 
Jun 13 16:12:16 rogue-thor app/heroku-redis: source=REDIS sample#active-connections=1 sample#load-avg-1m=0.13 sample#load-avg-5m=0.155 sample#load-avg-15m=0.12 sample#read-iops=0 sample#write-iops=0 sample#memory-total=15664360kB sample#memory-free=13104248kB sample#memory-cached=641888kB sample#memory-redis=2306848bytes sample#hit-rate=0.6769 sample#evicted-keys=0 
```

**Comment by sbsmith86** [comment]
Cool! let's try chunking the query

**Pull request merged**